src/portal/lib/src/create-edit-endpoint/create-edit-endpoint.component.html

```diff
@@ -41,7 +41,7 @@ <h3 class="modal-title">{{modalTitle}}</h3>
           <textarea type="text" class="inputWidth" row=3 name="description" [(ngModel)]="target.description"></textarea>
         </div>
         <!-- Endpoint Url -->
-        <div class="form-group">
+        <div class="form-group" [hidden]="target.type === 'dockerHub'">
```

Comment:
Some registry not support too, like huawei SWR.

Reply:
So how should I judge whether it supports it?

Reply:
We'll discuss this tonight in the Replication Workgroup meeting up.

Reply:
OK ,then this pr is temporarily not making changes here.